marquee/marquee.ino

```diff
@@ -38,9 +38,6 @@ void configModeCallback (WiFiManager *myWiFiManager);
 int8_t getWifiQuality();
 
 // LED Settings
-const int offset = 1;
-int refresh = 0;
-String message = "hello";
```

Comment:
These variables were unused, and there I'm just pulling them to make the code cleaner to read.